packages/vc-handler-ld-local/src/ld-credential-module.ts

```diff
@@ -55,18 +59,29 @@ export class LdCredentialModule {
     const suite = this.ldSuiteLoader.getSignatureSuiteForKeyType(key.type, key.meta?.verificationMethod?.type)
     const documentLoader = this.ldDocumentLoader.getLoader(context, true)
 
-    // some suites can modify the incoming credential (e.g. add required contexts)W
+    // some suites can modify the incoming credential (e.g. add required contexts)
     suite.preSigningCredModification(credential)
     debug(`Signing suite will be retrieved for ${verificationMethodId}...`)
     const signingSuite = await suite.getSuiteForSigning(key, issuerDid, verificationMethodId, context)
     debug(`Issuer ${issuerDid} will create VC for ${key.kid}...`)
-    const verifiableCredential = await vc.issue({
-      credential,
-      purpose,
-      suite: signingSuite,
-      documentLoader,
-      compactProof: false,
-    })
+
+    let verifiableCredential
+    //Needs to be signed using jsonld-signaures@5.0.1
+    if (key.type === 'Bls12381G2') {
+      verifiableCredential = await jsonld.sign(credential, {
```

Comment:
Cannot use it because vc-js uses different packages

require('@digitalcredentials/jsonld');
const jsigs = require('@digitalcredentials/jsonld-signatures');

Reply:
Yes? Those are forks compatible with RN